lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -783,8 +797,10 @@ private boolean achievedWithGpg45Profile(
                                 gpg45Scores, requestedVot.getSupportedGpg45Profiles())
                         : Optional.empty();
 
+        var isBreaching = ciMitUtilityService.isBreachingCiThreshold(contraIndicators, requestedVot);
```

Comment:
I think this logic may want to be inside gpg45ProfileEvaluator.getFirstMatchingProfile since we'll need to do something for evaluating scores at the end